technical/installation/deployment.md

```diff
@@ -381,10 +391,33 @@ controlled through [environment variables](https://docs.openwebui.com/getting-st
 The deployment comes with some default values that match the current AarhusAI setup, but you can change them
 to fit your needs in `values.yaml`.
 
-Also note, that `was-middleware.yaml` found in the openwebui templates folder should be updated to match your municipality's
-accessibility statement(s) (tilgængelighedserklæringer).
+Also note, that `was-middleware.yaml` found in the openwebui templates folder should be updated to match your
+municipality's accessibility statement(s) (tilgængelighedserklæringer).
+
+First you need to update the ingress in `applications/openwebui/values.yaml` to match your domain name. You also have to
+read through this file and update the settings to match your needs.
+
+**Note**: You need to allow "Sign-up" to allow the creation of the first user as _admin_ in the system (setting
+`ENABLE_SIGNUP` to `True`). You can make another deployment right after to disable the sign-up.
```

Comment:
This pops up in the middle of a section regarding installing `ingress`. Can we move somewhere else?